sklearn/ensemble/_hist_gradient_boosting/gradient_boosting.py

```diff
@@ -1310,6 +1303,14 @@ class HistGradientBoostingRegressor(RegressorMixin, BaseHistGradientBoosting):
         "quantile",
     )
 
+    _parameter_constraints = {**BaseHistGradientBoosting._parameter_constraints}
+    _parameter_constraints.update(
+        {
+            "loss": [StrOptions(set(_VALID_LOSSES)), BaseLoss],
+            "quantile": [Interval(Real, 0, 1, closed="both"), None],
+        }
+    )
+
```

Comment:
Sure, I agree that it is more readable. I think it does not occur in KMeans because it does not have `None` in the parameter constraint, but I didn't verify that.